src/main/java/org/folio/services/fund/FundDetailsService.java

```diff
@@ -109,8 +111,8 @@ public Future<List<ExpenseClass>> retrieveExpenseClasses(String fundId, String f
             .filter(expenseClass -> budgetExpenseClassIds.result().contains(expenseClass.getId()))
             .collect(toList()));
       })
-      .onSuccess(expenseClasses -> logger.debug("retrieveExpenseClasses:: found expense classes for fund id='{}', size={} ", fundId, expenseClasses.size()))
-      .onFailure(t -> logger.error("Retrieve expense classes for fund id='{}' failed", fundId, t));
+      .onSuccess(expenseClasses -> log.info("retrieveExpenseClasses:: found expense classes for fund id={}", fundId))
```

Comment:
its get request that can be invoked very often, maybe lets remain is as debug

Reply:
changed to debug, thanks